paper-ieee-trans.tex

```diff
@@ -63,61 +63,63 @@
 
-By analyzing the European Commission (EC)-funded software projects in the last decade
-we have observed a continuous increase of the prominence and robustness of
-testing and validation procedures. One of many reasons is related to the
+Analysing the European Commission (EC)-funded software projects over the last decade,
+one can observe a continuous increase in the prominence and robustness of software
+testing and validation procedures. One of many reasons, is related to the
 evolution of the software engineering methodologies (creation of the Agile
 Manifesto \cite{agile-manifesto}, and rise of DevOps practices \cite{zhu}) together
 with the parallel advancements in the information and communication technology
 field (such as virtualization, the enabling technology for Cloud computing and 
 operating--system--level virtualization known as containerization). All of which
 complemented by the emergence of automation and event-response tools.
 
-In this paper, we introduce a set of EC-funded projects
-that go from 2001 up to now, in which the authors of this summary were involved.
-For each project, we describe the challenges arised, the methodologies used and
-the achievements towards software reliability. The reminder of this paper is
+In this paper, we outline a set of EC-funded projects
+ranging from 2001 to now, in which the authors were involved.
+For each project, the challenges, the methodologies and
+the achievements towards software reliability are described.
+
+The reminder of this paper is
 organized as follows. Section \ref{sec:ev} details the evolution of research on
 software reliability in EC-projects. Section \ref{sec:ntsr} provides
 information about new research trends in software reliability based upon the 
-experiences from the recent {\sl INDIGO--DataCloud} project. Finally, Section \ref{sec:con} concludes.
+experiences from the recent {\sl INDIGO--DataCloud} project. Conclusions are drawn in Section \ref{sec:con}.
 
 \section{Evolution of research on software reliability in EC-projects}
 \label{sec:ev}
 
-The EC projects considered in this text are listed chronologically in Table
-\ref{tab:eup}. In most cases, each project paved the way of the subsequent projects,
-following a logic evolution in which the quality and reliability of software have
-increasingly had a more significant role. Experiences gathered during the 
-course of each project support this fact and, as it is shown in Table \ref{tab:feat},
-latest projects are the most committed in terms of software reliability awareness.
-Table \ref{tab:feat} summarizes also the topics addressed in the following sections, highlighting both
+The EC projects considered in this paper are listed chronologically in Table
+\ref{tab:eup}. Each project paved the way of the subsequent projects,
+following a logic evolution in which the role of the quality and reliability of the software have
+consistently increased. The experience gained during the 
```

Comment:
Probably remove the sentence "the experience gained during .." so to leave it as "As shown in table \ref{tab:feat}, the most recent projects are more committed and aware .."